snapshot/git/gitdb/checkout.go

```diff
@@ -51,7 +52,17 @@ func (db *DB) checkout(id snap.ID) (path string, err error) {
 		return db.checkoutFSSnapshot(v.SHA())
 	case kindGitCommitSnapshot:
 		// For GitCommitSnapshot's, we use dataRepo's work tree.
-		return db.checkoutGitCommitSnapshot(v.SHA())
+		if id == db.currentSnapID {
```

Comment:
We aren't using git_tags.go - it was created as a backup in case bundlestore wasn't working out. I'd like to delete it after we hit MVP.

We don't want to clean up the last run since we're trying to preserve the build cache for tasks with the same snapshotID.